app/javascript/packages/normalize-yaml/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@18f/identity-normalize-yaml",
+  "private": true,
+  "version": "1.0.0",
+  "type": "module",
+  "bin": {
+    "normalize-yaml": "./cli.js"
+  },
+  "dependencies": {
+    "prettier": "^2.2.1",
+    "smartquotes": "^2.3.2",
+    "yaml": "^2.0.0-5"
+  }
+}
```

Comment:
is there anything we can do that marks this as "please don't import me into the browser, I'm just meant for development"?

Reply:
> is there anything we can do that marks this as "please don't import me into the browser, I'm just meant for development"?

It's not something I've seen often. Usually it's a judgment call from the consumer, and bundlers would often go out of their way to support packages written for a Node setting to be bundled for a browser (e.g. [Webpack](https://webpack.js.org/configuration/node/)). This package _could_ run in a browser, AFAICT, even if the bundle size would be pretty large. If we really wanted to, we could set a [`browser` property](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#browser) here that points to a no-op, but I don't really think it's necessary.